src/pkg/packaging-tools/windows/product/product.wxs

```diff
@@ -49,6 +49,9 @@
     <Directory Id="TARGETDIR" Name="SourceDir">
       <Directory Id="$(var.Program_Files)">
         <Directory Id="DOTNETHOME" Name="dotnet" />
+        <?ifdef DefineDefaultInstallationDir ?>
+          <Directory Id="PROGRAMFILES_DOTNET" Name="dotnet" />
+        <?endif?>
```

Comment:
I would prefer adding this as an unconditional `Fragment` in `src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs`: this directory doesn't have a clear meaning in context, and it makes this highly shared file more complex. For example, over here:

https://github.com/dotnet/core-setup/blob/b44e5a25b376cfa6e0581cd69af05ce655d72f40/src/pkg/packaging-tools/windows/product/breadcrumbstorefolder.wxs#L28-L37

IMO this is fine to follow up on later, though, for the sake of getting something that we know works merged.

Reply:
@dagood I went ahead and made your recommended change and re-tested. I'll merge when CI is green.